api/src/main/java/org/openmrs/ProgramWorkflowState.java

```diff
@@ -20,14 +33,23 @@
 	// Properties
 	// ******************
 	
+	@Id
+	@Column(name = "program_workflow_state_id")
+	@GeneratedValue(strategy = GenerationType.AUTO)
```

Comment:
@alinmihaila99, can you run the `ValidateHibernateMappingsDatabaseIT` test locally. You might want to change the generation type to IDENTITY.

Reply:
I used the generation type IDENTITY in my second commit, but the travis failed. I also ran the ValidateHibernateMappingsDatabaseIT test locally and it passed.